tests/pytests/standard_suite_runner.py

```diff
@@ -59,6 +59,8 @@ def runner_asserter(inp, subject, method, basis, tnm):
     scf_type = natural_values[scf_type]
 
     atol = 1.0e-6
+    if driver == "hessian":
+        atol = 2.0e-6  # todo more elaborate
```

Comment:
Please elaborate on this comment to be meaningful to others.

Reply:
done. there's an elaborate e/g/h absolute and relative tolerance criteria implemented in qcdb and qcng that hasn't been ported to psi4 yet.